test/server/test-run-error-formatting-test.js

```diff
@@ -366,5 +366,21 @@ describe('Error formatting', () => {
         it('Should test messages for all error codes', () => {
             expect(untestedErrorTypes).to.be.empty;
         });
+
+        it('Errors codes should be unique', () => {
+            function getDuplicates (codes) {
```

Comment:
```js
function hasDuplicates(codes) {
    return new Set(codes).size !== codes.length;
}
```

Reply:
The current way allows to [observe](https://github.com/DevExpress/testcafe/pull/3430/files#diff-b2cea1e5219560251f905e9d2ac83d12R381) the found duplicates.

Reply:
Ok